runway/data_types.py

```diff
@@ -70,8 +76,8 @@ def setup(opts):
     :type description: string, optional
     """
 
-    def __init__(self, name='field', description=None):
-        super(any, self).__init__('any', name=name, description=description)
+    def __init__(self, description=None):
+        super(any, self).__init__('any', description=description)
```

Comment:
With the `name` property removed, I opted to explicitly pass the description kwarg for clarity, but if you would prefer `**kwargs` @agermanidis I'm ok with doing that instead.